docx/text/paragraph.py

```diff
@@ -42,10 +41,11 @@ def add_run(self, text=None, style=None):
         return run
 
     def add_sdt(self, tag_name, text='', alias_name='', temporary='false', locked='unlocked',
-                placeholder_txt=None, style='Normal', bold=False, italic=False, type=SdtType.RICH_TEXT):
+                placeholder_txt=None, style='Normal', bold=False, italic=False):
         """
-        Adds new Structured Document Type ``w:sdt`` field to the Paragraph element.
+        Adds new Structured Document Type ``w:sdt`` (Plain Text Content Control) field to the Paragraph element.
         """
+        from docx.sdt import SdtBase
 
```

Comment:
@aleksandarbos I have circular dependency error  and to resolve it I have removed type from add_sdt since it's unused and moved SdtBase import to inside of the function
Note: locked is also unused